doc/install.rst

```diff
@@ -0,0 +1,49 @@
+.. _install:
+
+==================
+Installation guide
+==================
+
+
+StochDynamicProgramming installation
+------------------------------------
+
+To install StochDynamicProgramming::
+
+    julia> Pkg.clone("https://github.com/leclere/StochDynamicProgramming.jl")
+
+Once the package is installed, you can import it in the REPL::
+
+    julia> using StochDynamicProgramming
+
+
+Install a linear programming solver
+-----------------------------------
+
+SDDP need a linear programming solver to run.
+
+You could refer to the documentation of JuMP_ to get a solver and interface it with SDDP.
+
```

Comment:
I think Clp is automatically installed when our package is being installed, thanks to the REQUIRE file